python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -294,9 +294,13 @@ def generate_subsystems_doc():
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
-                            cf.write(
-                                f"  * :any:`{state.split('_')[1]} <lsst.ts.idl.enums.{subsystem}.{state_name}.{state.split('_')[1].upper()}>`\n"  # noqa
-                            )
+                            # state format starts as [\n statename_state ]
+                            state = state.split("_")[
+                                1:
+                            ]  # Some states have more than one underscore split.
+                            state = "".join(state)  # Make that list a string
```

Comment:
Please change lines 298 - 301 to:
```
state = state.split("_", 1)[1]
```
there is no need to do more than a single split, and having split at only the first "_" you don't have to re-assemble anything.

Also please consider `enum_item` instead of `state` as the variable name and "enum_entry" instead of "state" for the thing you are splitting.